dot/rpc/modules/author_test.go

```diff
@@ -14,173 +14,145 @@ import (
 	"github.com/ChainSafe/gossamer/dot/types"
 	"github.com/ChainSafe/gossamer/internal/log"
 	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/ChainSafe/gossamer/lib/crypto/ed25519"
 	"github.com/ChainSafe/gossamer/lib/crypto/sr25519"
 	"github.com/ChainSafe/gossamer/lib/keystore"
-	"github.com/ChainSafe/gossamer/lib/runtime"
-	"github.com/ChainSafe/gossamer/lib/runtime/wasmer"
 	"github.com/ChainSafe/gossamer/lib/transaction"
-	"github.com/google/go-cmp/cmp"
+
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/mock"
 	"github.com/stretchr/testify/require"
 )
 
-func TestAuthorModule_HasSessionKey_WhenScaleDataEmptyOrNil(t *testing.T) {
-	keys := "0x00"
-	runtimeInstance := wasmer.NewTestInstance(t, runtime.NODE_RUNTIME)
-
-	coremockapi := new(apimocks.CoreAPI)
-
-	decodeSessionKeysMock := coremockapi.On("DecodeSessionKeys", mock.AnythingOfType("[]uint8"))
-	decodeSessionKeysMock.Run(func(args mock.Arguments) {
-		b := args.Get(0).([]byte)
-		dec, err := runtimeInstance.DecodeSessionKeys(b)
-		decodeSessionKeysMock.ReturnArguments = []interface{}{dec, err}
-	})
-
-	module := &AuthorModule{
-		coreAPI: coremockapi,
-		logger:  log.New(log.SetWriter(io.Discard)),
-	}
-
-	req := &HasSessionKeyRequest{
-		PublicKeys: keys,
-	}
-
-	var res HasSessionKeyResponse
-	err := module.HasSessionKeys(nil, req, &res)
-	require.NoError(t, err)
-	require.False(t, bool(res))
-
-	coremockapi.AssertCalled(t, "DecodeSessionKeys", mock.AnythingOfType("[]uint8"))
-}
-
-func TestAuthorModule_HasSessionKey_WhenRuntimeFails(t *testing.T) {
-	coremockapi := new(apimocks.CoreAPI)
-	coremockapi.On("DecodeSessionKeys", mock.AnythingOfType("[]uint8")).Return(nil, errors.New("problems with runtime"))
+func TestAuthorModule_HasSessionKeys(t *testing.T) {
+	testReq := "0xd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d34309a9d2a24213896ff06895db16" +
```

Comment:
nit make stuff constant whenever possible

```suggestion
	const testReq = "0xd43593c715fdd31c61141abd04a99fd6822c8558854ccde39a5684e7a56da27d34309a9d2a24213896ff06895db16" +
```